Topics/Development_Process/Shell/intro_to_shell.md

```diff
@@ -0,0 +1,159 @@
+# Detailed Introduction to Shell Commands
+
+Shell commands are the primary interface for users to interact with an operating system's services in a Unix or Unix-like environment. These commands are entered into a command-line interface (CLI), such as Bash in Linux or Command Prompt in Windows.
```

Comment:
Maybe you want to mention that the default shell in MacOS is `zsh`

Reply:
Thank you for pointing out! Already updated default shell in MacOS is `zsh`.